docs/1.0. Certificate Provisioning.md

```diff
@@ -47,67 +47,79 @@ The entity that is providing the EST API in accordance with [RFC 7030][RFC-7030]
+1. When the NMOS Client (EST Client) is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+2. The NMOS Client will then request the Root CA for the target network, from the EST Server.
+   - The NMOS Client can assume the EST Server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
+   - The NMOS Client uses the Root CA returned to secure further communications with the EST Server and the NMOS Servers.
 
-The workflow if no valid manufacturer TLS Client Certificate is present or if EST Client is moved to a new network are covered in detail in the following [sections](#est-client).
+The workflow if the NMOS Client is moved to a new network is covered in detail in the [EST Client Behaviour](#est-client-behaviour) section.
 
-![NMOS EST WorkFlow](images/nmos_est_workflow.png)
+![NMOS Client Workflow](images/nmos_client_workflow.png)
+## NMOS Server Automated Certificate Provisioning Flow (informative)
+
+To enable zero-configuration TLS Certificate provisioning, NMOS implementations MUST include a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority. Manufacturers MUST provide the corresponding Root Certificate Authority public key to customers, to be loaded into the EST Server. If the TLS Client Certificate or the chain of trust is compromised, the manufacturer MUST revoke the comprised certificates.
+
+1. Before the NMOS Server (EST Client) is shipped from the factory it must be provisioned with a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority.
+2. When the NMOS Server is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+3. The NMOS Server will then request the Root CA for the target network, from the EST Server.
+   - The NMOS Server can assume the EST Server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
+   - The NMOS Server uses the Root CA returned to secure further communications with the EST Server.
+4. The NMOS Server will generate a Certificate Signing Request (CSR) for all supported digital signature algorithms with appropriate key lengths.
+   - Ensure the DNS resolvable name for the current domain of the NMOS Server is included in the CSR.
+   - Use appropriate values for the remaining CSR fields.
+5. The NMOS Server will send each CSR to the EST Server.
+   - The NMOS Server will provide the manufacturer-issued certificate during the TLS handshake, if this certificate is valid.
+   - The EST Server will use the TLS Client Certificate to authenticate and then authorize the NMOS Server. If the NMOS Server meets the local CA's security policy, it will be issued with a TLS Server Certificate.
```

Comment:
Does "local CA" something different than "Root CA [of the current network]" used elsewhere?

Reply:
Perhaps this should not refer the the CA and instead refer to the EST Server
```suggestion
   - The EST Server will use the TLS Client Certificate to authenticate and then authorize the NMOS Server. If the NMOS Server meets the EST Server's security policy, it will be issued with a TLS Server Certificate.
```